forage-android/src/main/java/com/joinforage/forage/android/ForageSDK.kt

```diff
@@ -134,10 +148,21 @@ class ForageSDK : ForageSDKInterface {
                 httpUrl = ForageConstants.provideHttpUrl(),
                 logger = logger
             ),
-            logger = logger
+            logger = logger,
+            responseMonitor = measurement
         ).checkBalance(
             paymentMethodRef = paymentMethodRef
         )
+        measurement.end()
+
+        val outcome = if (response is ForageApiResponse.Failure) {
+            EventOutcome.FAILURE
```

Comment:
Just reading through the comments to get up to speed, but did we also want to add the `forage_error_code` here?

Reply:
So I actually was setting the value within the function call above... but this is a better spot! I'm going to update.